Assignment 5.Rmd

```diff
@@ -66,7 +66,7 @@ Upload the data "intelligent_tutor_new.csv". This is a data set of a differnt sa
 ```{r}
```

Comment:
Do you have an answer to the above question? What inferences can you make from your tree?